frontend/jce-app/src/views/profile.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+
+function Profile(props) {
+  useEffect(() => {
+    document.title = props.title;
+  }, [props.title]);
+  return (
+    <div style={{ margin: "20px" }}>
+      <h1 className="section-jce">Profile</h1>
+
+      <div style={{ margin: "20px" }}>
+        <h2 className="header-jce">Personal Information</h2>
+
+        <div className="content mb-4">
+          <div className="row">
+
+            <div className="col-4">
+              <h5 className="general-jce">Name</h5>
+              <input className="w-100" type="text" id="name" name="name"readOnly/>
```

Comment:
You could also use the style "general-jce" in all the inputs to have the same format of the labels